structs.go

```diff
@@ -312,41 +312,28 @@ type Presence struct {
 	Since  *int     `json:"since"`
 }
 
+// A game type
+type GameType int
+
+const (
+	GameTypeGame GameType = iota
+	GameTypeStreaming
+)
+
 // A Game struct holds the name of the "playing .." game for a user
 type Game struct {
-	Name string `json:"name"`
-	Type int    `json:"type"`
-	URL  string `json:"url,omitempty"`
+	Name string   `json:"name"`
+	Type GameType `json:"type"`
+	URL  string   `json:"url,omitempty"`
 }
 
 // UnmarshalJSON unmarshals json to Game struct
 func (g *Game) UnmarshalJSON(bytes []byte) error {
```

Comment:
This method shouldn't be needed anymore.

Reply:
Very true. Will delete it!